code/modules/events/radiation_storm.dm

```diff
@@ -21,8 +21,10 @@
 		/area/vox_trading_post,
 	)
 
-/datum/event/radiation_storm/can_start()
-	return 50
+/datum/event/radiation_storm/can_start(var/list/active_with_role)
+	if(active_with_role.len > 6)
```

Comment:
this is a snowflaked list that only has certain types of jobs in it

a crew of greyshirts, chefs, barkeeps, pod people and mice and <6 "special" jobs won't ever see radstorms then

why not try the get_active_player_count global proc, that will give you the expected result

Reply:
reading the comment above number_active_with_role (this is why comments suck), the entire proc is outdated

// Returns how many characters are currently active(not logged out, not AFK for more than 10 minutes)

i'd bandaid this by having any roles that fall through the proc be assigned to a "Other" job, then it's usable again and even fixes carps not showing up unless theres >6 snowflake roles

Reply:
i think engineering/medical/secborgs are counted twice, too...

Reply:
maybe it's time to take an entire step back. looks to me like that list always has 8 entries, since its really just a map holding counters for departments. so my guess is this pr will do nothing, just like carp_migration that uses it is borked

Reply:
Good point. I looked it over and ``active_with_role`` seems to focus on jobs that are more relevant during events in general. Like, botanist is specifically listed. Can only imagine what events they'd be ideal to fight against (VINES). This PR looks like a half baked admin idea at this point so I'm chatting with some of the premo deadpoppers on Handled on Discord right now to see what might improve the situation.

Like I could swap this with requires a SINGLE medical person, and the PR is kind of the same in terms of deadpop (who plays a doc during deadpop?)

Reply:
See the sister PR on how cursed this wound up being. It's fixed now.